### PR TITLE
Configuring a project. Property files

### PR DESCRIPTION
**Pull request by IgChele**
Need some feedback on current information.

### CODE REVIEW

File: TL/18_Configuring_a_project._Property_files/Configuring_a_project.Property_files_README.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+            registerUrl = properties.getProperty("registerUrl");
+            overviewUrl = properties.getProperty("overviewUrl");
+            baseUrl = properties.getProperty("baseUrl");
+
+        } catch (IOException e) {
+            logger.error("Error loading properties file: " + e.getMessage());
+        }
+    }
+
+    public static String getBrowser() {
```
Comment (endalexei) [inline TL/18_Configuring_a_project._Property_files/Configuring_a_project.Property_files_README.md:256]: if here we read property from file i think a name like initProperty or something like that will be more appropriate by java naming convention. Name should reflect the action inside

**Review by endalexei — APPROVED** [review]
Good job Igor, beside that small comment, otherwise it is what I expected to see.

File: TL/18_Configuring_a_project._Property_files/Configuring_a_project.Property_files_README.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+import org.apache.logging.log4j.Logger;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class PropertiesManager {
+    private static final Logger logger = LogManager.getLogger(PropertiesManager.class);
+    private static String browser;
+    public static String homeUrl;
```
Comment (Liudmila-Topal) [inline TL/18_Configuring_a_project._Property_files/Configuring_a_project.Property_files_README.md:235]: Why are you using different access modifiers?

**Comment by bodacd** [comment]
Great Job, please add your name in readme file as an author and the PR will be ready to be merged

**Pull request merged**